include/net/socket.h

```diff
@@ -797,6 +797,13 @@ static inline char *inet_ntop(sa_family_t family, const void *src, char *dst,
 #define EAI_FAMILY DNS_EAI_FAMILY
 #endif /* defined(CONFIG_NET_SOCKETS_POSIX_NAMES) */
 
+#define IFNAMSIZ 64
```

Comment:
Just checked and the maximum device name length is 48 bytes `Z_DEVICE_MAX_NAME_LEN`, should we use that value here as it is rather pointless to use longer length.
So we could have `#define IFNAMSIZ MIN(Z_DEVICE_MAX_NAME_LEN, 64)`
Or is the value dictated in some spec, any idea?

Reply:
It was just an arbitrary value, it makes sense to use `Z_DEVICE_MAX_NAME_LEN`. I wonder if we need to use `MIN` though, is `Z_DEVICE_MAX_NAME_LEN` a subject of change? I'd rather say to use it directly, to keep i tsimple.

Reply:
Sure we can use the value directly, I was just assuming that the 64 was mentioned in some spec.